Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -559,52 +559,71 @@ section ContinuousMul
 variable [ContinuousMul α]
 
 @[to_additive]
-theorem tprod_mul (hf : Multipliable f) (hg : Multipliable g) :
+protected theorem Multipliable.tprod_mul (hf : Multipliable f) (hg : Multipliable g) :
     ∏' b, (f b * g b) = (∏' b, f b) * ∏' b, g b :=
   (hf.hasProd.mul hg.hasProd).tprod_eq
 
+@[to_additive, deprecated (since := "2025-04-11")] alias tprod_mul := Multipliable.tprod_mul
```

Comment:
This way the additive version isn't deprecated. Unfortunately, `to_additive (attr := deprecated (since := ...))]` doesn't work either. My workaround is to use something like
```suggestion
@[deprecated (since := "2025-04-11")] alias tsum_add := Summable.tsum_add
@[to_additive existing, deprecated (since := "2025-04-11")] alias tprod_mul := Multipliable.tprod_mul
```

Reply:
doh!

what is wrong with just two lines for each lemma? 

```
@[deprecated (since := "2025-04-11")] alias tsum_add := Summable.tsum_add
@[deprecated (since := "2025-04-11")] alias tprod_mul := Multipliable.tprod_mul
```

Reply:
It registers the alias with the `to_aditive` machinery.

Reply:
Ok - I believe I have fixed all these now.